contrib/bdii/htcondor-ce-provider

```diff
@@ -208,6 +216,11 @@ def main():
         except (KeyError, RuntimeError):
             state = 'critical'
             state_info = 'Authorization ping failed'
+        except Exception, e:
+            sys.stderr.write("Ping to CE schedd on %s timed out.\n" % ce_host )
```

Comment:
```suggestion
            sys.stderr.write("Ping to CE schedd on %s timed out after 10s.\n" % ce_host )
```

Reply:
Doing this change with the new time_out variable